hack/testing/test-367-logforwarding.sh

```diff
@@ -47,8 +47,27 @@ else
   deploy_elasticsearch_operator
 fi
 
-os::log::info "Deploying cluster-logging-operator"
-deploy_clusterlogging_operator
-exit 0
-os::log::info "Staring test of logforwarding"
-ELASTICSEARCH_IMAGE=quay.io/openshift/origin-logging-elasticsearch5:latest go test -parallel=1 ./test/e2e/logforwarding  | tee -a $ARTIFACT_DIR/test.log
\ No newline at end of file
+failed=0
+for dir in $(ls -d ./test/e2e/logforwarding/*); do
+  os::log::info "==========================="
+  os::log::info "Staring test of logforwarding $dir"
+  os::log::info "==========================="
+  os::log::info "Deploying cluster-logging-operator"
+  deploy_clusterlogging_operator
+  artifact_dir=$ARTIFACT_DIR/$(basename $dir)
+  mkdir -p $artifact_dir
```

Comment:
note that `mkdir -p $artifact_dir` will fail if `$artifact_dir` exists - which is fine for the CI environment - but might trip up someone running this outside of the CI environment - I suggest using `if [ ! -d $artifact_dir ] ; then mkdir -p $artifact_dir ; fi` to be safe

Reply:
Good point. Let's add that in a subsequent PR, so at least we can get CI working as soon as possible.

Reply:
what about `mkdir -p $artifact_dir || true` ? :smile:

Reply:
that will ignore all errors, including ones that you want to be reported e.g. permissions, os problems

Reply:
sure - go ahead and merge as-is
/lgtm